### PR TITLE
Onimaru shield and Fang dash should be done by clicking the space key

### PR DESCRIPTION
**Pull request by JoanMarcBardes**

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-p3d1n3 Change keyboard first ability](https://app.clickup.com/t/p3d1n3)

File: Penteract/Assets/Scripts/Player.cpp [hunk 1]
```diff
@@ -171,7 +171,7 @@ bool Player::GetInputBool(InputActions action, bool useGamepad) {
 		if (useGamepad && Input::IsGamepadConnected(0)) {
 			return Input::GetControllerAxisValue(Input::SDL_CONTROLLER_AXIS_TRIGGERLEFT, 0) > PRESSED_TRIGGER_THRESHOLD;
 		} else {
-			return Input::GetMouseButtonDown(2) || Input::GetMouseButtonRepeat(2);
+			return Input::GetKeyCodeDown(Input::KEYCODE::KEY_Q) || Input::GetKeyCodeRepeat(Input::KEYCODE::KEY_SPACE);
```
Comment (devildrake) [inline Penteract/Assets/Scripts/Player.cpp:174]: Shouldn't this say KEYCODE::KEY_SPACE in both cases?

**Review by Jordi-Gil — DISMISSED** [review]

File: Penteract/Assets/Scripts/Player.cpp (hunk 1 above)
Comment (Jordi-Gil) [reply Penteract/Assets/Scripts/Player.cpp:174]: Yes

**Review by Jordi-Gil — CHANGES_REQUESTED** [review]

File: Penteract/Assets/Scripts/Player.cpp (hunk 1 above)
Comment (JoanMarcBardes) [reply Penteract/Assets/Scripts/Player.cpp]: yes i dont know why it is key_q (i put key_space in both)

**Review by jcarlos0305 — APPROVED** [review]

**Review by Jordi-Gil — APPROVED** [review]

**Review by Yonimevicio — APPROVED** [review]

**Pull request merged**